sway/config/output.c

```diff
@@ -351,6 +356,22 @@ static int compute_default_scale(struct wlr_output *output) {
 	return 2;
 }
 
+/* Lists of formats to try, in order, when a specific render bit depth has
+ * been asked for. The last format in the list should always be XRGB8888,
+ * as a reliable backup in case the others are not available. */
+static const uint32_t *bit_depth_preferences[] = {
+	[RENDER_BIT_DEPTH_8] = (const uint32_t []){
+		DRM_FORMAT_XRGB8888,
+		0
```

Comment:
Nit: use `DRM_FORMAT_INVALID` to make it more explicit that this magic value doesn't refer to a valid format

Reply:
Done.